src/Routes/ImageManager/steps/packages.js

```diff
@@ -3,19 +3,14 @@ import componentTypes from '@data-driven-forms/react-form-renderer/component-typ
 import { Text } from '@patternfly/react-core';
 
 export default {
-  title: 'Packages',
+  title: 'Additional Packages',
   name: 'packages',
   nextStep: 'review',
   fields: [
     {
       component: componentTypes.PLAIN_TEXT,
       name: 'google-cloud-text-component',
-      label: (
-        <Text>
-          Add optional additional packages to your image by searching available
-          packages.
-        </Text>
-      ),
+      label: <Text>Add packages to your RHEL 8.3 image.</Text>,
```

Comment:
We might want to have dynamic label here, based on what was selected in image output step.

Reply:
DONE